Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -535,7 +539,9 @@ private void ThreadLoadPatcherData()
                 var inputArgumentsPatcherDataReader = new InputArgumentsPatcherDataReader();
                 _data.Value = inputArgumentsPatcherDataReader.Read();
 #endif
-                DebugLogger.LogVariable(_data.Value.AppSecret, "Data.AppSecret");
+                AppSecret = _data.Value.AppSecret;
+                BuildTraceableAppSecret(AppSecret);
```

Comment:
Instead of building it here, I would propose a lazy evaluation inside `TraceableAppSecret` property to make this value independent from code instructions flow.

An article that explains the idea of lazy evaluation - https://dev.to/fluffynuts/lazy-props-for-the-lazy-3i6o

Reply:
Done